### DISCUSSION TITLE
Como vocês controlam o financeiro da PJ?

### DISCUSSION THREAD

**Discussion by igorlvicente** [DyegoCosta/trabalhando-remoto#123] [General]
Usam algum SaaS como Conta Azul ou quickbooks?
Fazem na planilha? Se sim, qual model utilizam?

**Comment by marcelolima** [comment]
Planilha, utilizo essa pra Lucro Presumido criada pelo @lcguida : https://github.com/DyegoCosta/trabalhando-remoto/issues/87 

No meu caso, os dados tão todos na Contabilizei também, mas prefiro fazer esse controle por fora pra cruzar os dados no final (não confio 100% na Contabilizei)

**Comment by igorlvicente** [comment]
Descobri, há alguns dias, sobre o GnuCash. Ele tem uma documentação boa e estou até aprendendo alguns conceitos contábeis bem úteis. No futuro posto aqui como me sai com ele.
